### ISSUE TITLE
Create Inbox Reborn Launcher?

### ISSUE THREAD

**Issue by SharonPerp** [boukestam/inbox-in-gmail#84]
This is kind of out there, but I used to use this great little App Launcher for Inbox. It kept my email separate from my usually huge number of Chrome tabs. And, I could close Chrome when it sucked down all my resources without disturbing Inbox.

Looked just like regular Inbox but opened as if it were it's own app. And, I believe it technically showed up as a Chrome App.

Amazing job on what you've done so far, thanks!

**Comment by russelldc** [comment]
@SharonPerp Interesting idea, but unless I'm misunderstanding, you're describing a Chrome app, which Google discontinued a couple years ago. [See announcement here](https://blog.chromium.org/2016/08/from-chrome-apps-to-web.html)

One way to achieve what you're asking for is to create an [Electron](https://electronjs.org/) app that points to the Gmail website, but somehow includes this extension.

**Comment by SharonPerp** [comment]
Actually, I decided to take a look, and it's still in the Chrome store: https://chrome.google.com/webstore/detail/app-launcher-for-inbox-by/ajjmenfgmbpceneffopmikhahnkeebje?hl=en

Of course, now it launches the ugly gmail interface in its own window instead of actual Inbox.
Hope that helps!

**Comment by russelldc** [comment]
@SharonPerp Yup, that one was last updated on **June 30, 2015**, long before Google discontinued Chrome Apps. The previously published ones were supposed to be removed from the Chrome web store as of last year, but that date got pushed back a bit more.

**Comment by SharonPerp** [comment]
Hmmm, well, I don't understand how apps work really, all I know is it was fixed pretty quickly whenever Google changes broke it, and was perfectly useable until just recently.
